sdk/system/grub.go

```diff
@@ -37,7 +37,7 @@ func setGRUBOptions(opts map[string]string) error {
 	}()
 
 	for k, v := range opts {
-		out, err := utils.SH(fmt.Sprintf(`grub2-editenv /tmp/oem/grubenv set "%s=%s"`, k, v))
+		out, err := utils.SH(fmt.Sprintf(`%s /tmp/oem/grubenv set "%s=%s"`, machine.FindCommand("grub2-editenv", []string{"grub2-editenv", "grub-editenv"}), k, v))
```

Comment:
some distros have grub-install, or grub-editenv instead of grub2-editenv and grub2-install. Hence some logic to detect that. note we can't drop symlinks still due to #435